README.md

```diff
@@ -65,6 +65,20 @@ gluster_mount 'volume_name' do
 end
 ```
 
+#### Parameters
+
+- `server` - The primary server to fetch the volfile from. Required.
+
+- `backup_server` - Backup servers to obtain the volfile from. Required.
```

Comment:
> `Required` 

Not per the [LWRP resource](https://github.com/joyofhex/chef-gluster/blob/mount_owner/resources/mount.rb#L29)